### ISSUE TITLE
[Bug] Chart - omitting a data point in a LineSeries crashes the app on iOS

### ISSUE THREAD

**Issue by EddyVerbruggen** [telerik/nativescript-ui-feedback#188]
When rendering a `LineSeries` and leaving out a data point, the app crashes. In this example the value for may 14th is missing. So the date is defined, but the value is not.

### Which platform(s) does your issue occur on?
- iOS

On Android the graph is rendered fine:

<img width="395" alt="screen shot 2017-05-17 at 10 18 30" src="https://cloud.githubusercontent.com/assets/1426370/26144837/b5d0b6be-3aea-11e7-964b-cadefe54853d.png">

### Versions
- **Progress NativeScript UI version:** 2.0.1
- **CLI:** 3.0.1
- **Cross-platform modules:** 3.0.0
- **Runtime(s):** 3.0.0

### Please tell us how to recreate the issue in as much detail as possible.
Add a graph like this to an Angular app (not that this is Angular related):

```typescript
  export class WeightPlot {
    public Date: Date;
    public Weight?: number;
  }

  categoricalSource: Array<WeightPlot> = [
    { Date: new Date(2017, 4, 6), Weight: 84 },
    { Date: new Date(2017, 4, 10), Weight: 80.5 },
    { Date: new Date(2017, 4, 11), Weight: 82 },
    { Date: new Date(2017, 4, 12), Weight: 81 },
    { Date: new Date(2017, 4, 13), Weight: 79.5 },
    { Date: new Date(2017, 4, 14) }  // note that the Weight property is missing here
  ];
```

```html
  <RadCartesianChart tkExampleTitle tkToggleNavButton height="70%">
    <DateTimeCategoricalAxis tkCartesianHorizontalAxis dateFormat="dd/MM"></DateTimeCategoricalAxis>
    <LinearAxis tkCartesianVerticalAxis></LinearAxis>
    <LineSeries tkCartesianSeries [items]="categoricalSource" categoryProperty="Date" valueProperty="Weight"></LineSeries>
  </RadCartesianChart>
```

### Crashlog
I'd love to dig in deeper, but the crash seems to occur in the native UI for iOS classes which I don't have access to. The only thing Xcode tells me is:

```
2017-05-17 10:10:50.560 nativescript[39018:1637872] -[NSNull center]: unrecognized selector sent to instance 0x105905180
```

**Comment by ginev** [comment]
@EddyVerbruggen, can you try setting Weight to undefined: Weight: undefined and see if it helps?

**Comment by EddyVerbruggen** [comment]
I did that before opening this issue but it didn't help.

**Comment by basterbrugge** [comment]
I can confirm that it doesn't work with null nor undefined. Any updates on this?

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-chart@3.9.1. Enjoy!
